app/models/sam_account_reckoner.rb

```diff
@@ -31,4 +45,16 @@ def sam_status
   def client
     @client ||= Samwise::Client.new(api_key: DataDotGovCredentials.api_key)
   end
+
+  def vendor_summary
+    @vendor_summary ||= client.get_vendor_summary(duns: user.duns_number)
+  end
+
+  def duns_is_in_sam?
+    vendor_summary[:in_sam]
```

Comment:
assume this json returns a string val -- maybe we should check for equality here just in case?

eg: if it were `vendor_summary[:in_sam] == 'false' the current logic would return `true` because a string is "truthy"